src/Components/Navbar/index.js

```diff
@@ -0,0 +1,70 @@
+import React, { useState, useEffect } from "react";
+import "./static/css/navbar.css";
+import logo from "./static/images/agric.png";
+import { FaSearch} from "react-icons/fa";
+import {IoIosArrowRoundForward} from 'react-icons/io'
+import styled from "styled-components";
+import {BrowserRouter as Router,Switch,Route, Link } from "react-router-dom";
+
+let Search = styled.div`
```

Comment:
move this styled component to a file called styles in this folder and import it from there